cmd/kubeadm/app/cmd/init.go

```diff
@@ -353,9 +366,25 @@ func newInitData(cmd *cobra.Command, args []string, options *initOptions, out io
 		ignorePreflightErrors: ignorePreflightErrorsSet,
 		externalCA:            externalCA,
 		outputWriter:          out,
+		uploadCerts:           options.uploadCerts,
```

Comment:
Please make init data fail if externalCA and uploadCerts

Reply:
Thanks @fabriziopandini, I'll address this and finish the tests.